prov/efa/src/rxr/rxr_pkt_entry.c

```diff
@@ -349,7 +352,7 @@ ssize_t rxr_pkt_entry_inject(struct rxr_ep *ep,
 	/* currently only EOR packet is injected using shm ep */
 	peer = rxr_ep_get_peer(ep, addr);
 	assert(peer);
-	assert(rxr_env.enable_shm_transfer && peer->is_local);
+	assert(ep->use_shm && peer->is_local);
```

Comment:
I am not following this assert (or even the existing one). We can get into `rxr_pkt_post_ctrl_once()` even for a non-local peer right?

Reply:
Not sure I understand the comments. This is in rxr_pkt_entry_inject(), currently we only do inject for shm ep.